lib/puppet/pops/serialization.rb

```diff
@@ -7,12 +7,14 @@ def self.not_implemented(impl, method_name)
   class SerializationError < Puppet::Error
   end
 
-  PCORE_TYPE_KEY = '__pcore_type__'.freeze
+  PCORE_TYPE_KEY = '__ptype'.freeze
+  PCORE_TYPE_KEY_OLD = '__pcore_type__'.freeze
```

Comment:
This doesn't seem to be used. Was the intention here to be backward compatible? If so, that code must be added. If not, then this should be removed.

Reply:
should be removed